person.rb

```diff
@@ -0,0 +1,26 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age
+    @age >= 18
+  end
+
+  def can_use_services?
+    @parent_permission == true
+  end
+end
```

Comment:
- [ ] As per the project [requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md), `Public method can_use_services? that returns true if person is of age or if they have permission from parents.`, Public method can_use_services? that returns true if person is of age or if they have permission from parents. Something like so: 

```ruby
 def can_use_services? 
    of_age?() || @parent_permission
  end
```
Please kindly fix that.

Reply:
Tasks completed. Thank you.